docs/Test.md

```diff
@@ -0,0 +1,76 @@
+Centaur is an integration testing suite for the [Cromwell](http://github.com/broadinstitute/cromwell) execution engine.  It's purpose is to exercise the functionality of a specific deployment of Cromwell, to ensure that it is functioning properly 'in the wild'.  
+
+## Prerequisites
+
+Centaur expects to find a Cromwell server properly configured and running in server mode, listening on port 8000.  This can be configured by modifying the `cromwellUrl` parameter in `application.conf`.
+
+## Running
+
+There are two ways to invoke the intergration tests:
+* `sbt test` - compiles and run via sbt directly, simple but also has the problem of running two times cores tests in parallel which can overwhelm your Cromwell server if running in a development environment
+* `run_tests_parallel.sh [THREADS]` - runs the same tests with an enforced parallelism limit.  Defaults to `3` if not specified
+
+### Tags
+
+All tests are tagged with their name and their TESTFORMAT, and also any custom tags specified in the `.test` file.
+
+Tag names are all lower case, so a test named "tagFoo" has a tag "tagfoo".
+
+To run only those tests which have been tagged with a specified tag `tagFoo`:
+```
+sbt "testOnly * -- -n tagfoo"
```

Comment:
This looks like the old style invocation. It's completely different in the `centaur/README.MD` file that this PR is deleting...

Reply:
I think you must have copied this file from `cromwell-29`, but now we're deleting different content from `develop`? As @mcovarr pointed out, I think the file has been changed elsewhere too

Reply:
I copied the [Centaur README](https://github.com/broadinstitute/cromwell/blob/develop/centaur/README.md) which was taken from the [Centaur repo](https://github.com/broadinstitute/centaur/blob/develop/README.md). They appear to be identical to what's in the (renamed) docs/Centaur.md. What information are you not seeing?